src/tests/wallet_test.ts

```diff
@@ -1402,6 +1399,55 @@ describe("Wallet Class", () => {
       const seedlessWallet = Wallet.init(walletModel, "");
       expect(() => seedlessWallet.saveSeedToFile(filePath, false)).toThrow(Error);
     });
+
+    describe("#saveSeed additional coverage", () => {
```

Comment:
If I run into this docstring when a test fails this doesn't really tell me anything useful.

What is actual additional scenarios we're adding tests for?